packages/scss/src/components/_nav-side.scss

```diff
@@ -283,7 +283,6 @@
 			color: _component("navSide.compact-palette.alert-text");
 			padding: 0 .4em;
 			position: absolute;
-			right: 10px;
```

Comment:
In compact mod, this new should be below the text, centered and it should extend the "selection" box :)

Reply:
![image](https://user-images.githubusercontent.com/30234402/83515660-07db7180-a4d6-11ea-9229-b57d5e6b716b.png)